src/main/java/me/partlysanestudios/partlysaneskies/system/OneConfigScreen.java

```diff
@@ -934,6 +934,26 @@ public void resetBrokenStrings() {
     )
     public boolean bitShopOnlyShowAffordable = true;
 
+    @Dropdown(
+            category = "Economy",
+            name = "Coins to Cookies Preferred Currency",
+            description = "Select your preferred currency conversion for the /c2c command. Currencies are listed in alphabetical order. Default currency is USD. Once you've selected a currency, please restart your game.",
```

Comment:
You said you got it to work without needing to rs, right? If so you can remove the "restart your game thingy" ty

Reply:
oh fuck i forgot about that

give me a few hours

Reply:
_blazingly fast 🔥_

Reply:
> give me a few hours

*\*a few hours later\**

Reply:
Let him cook

Reply:
Omg ikr, how could you program on mc font. Like I get it's fixed pitch and fixed width, but use something other than minecraft font pleaseee

Reply:
fuck i completely forgot